pkg/metrics/build_info.go

```diff
@@ -0,0 +1,16 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+func registerVersionMetric(namespace string, version string) {
+	g := prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: namespace,
+		Name:      "build_info",
```

Comment:
Why not just `version`? Many other components use this name and I think we better have some common one.

Reply:
i had nothing to be expired by so i decided to go to the origins directly: https://github.com/prometheus/common/blob/2f0f5c4b6d958963f2a8d5447ad27ca7a931dcda/version/info.go#L38-L61 and https://github.com/prometheus/client_golang/blob/297fea317b926e4bb8d3c72022e172def2f68ed2/prometheus/metric.go#L89-L92

in other words, that is how prometheus dev would do such things _in my interpretation_. more strict args i would say sound like: const "build_info" is the only metric that is presented here (in the PR) and will be presented forever, it contains any needed build info but also it will be easy to expand it with any new build info cause build attrs are just labels and not smth that encoded in a metric name (comparing with `neofs_ir_version: 1`).
what additional info? well, even the issue that PR closes has two suggestion (`version` and `short_version`). i would also think about `go_version` (ok, we have another metric for that but i can imagine that one metric for everything may be more convenient), `build_time`, etc

Reply:
`GaugeVec` is OK and it can be extended if need be, it's just that we already use `version` for it in
 * nspcc-dev/neo-go#3009
 * nspcc-dev/neofs-http-gw#251
 * nspcc-dev/neofs-s3-gw#779

So I'd rather continue using it instead of introducing `build_info`. Nothing bad about the name itself except that we're already using a different one. And some third-party apps do that too, so it's not something surprisingly original here.

Reply:
really sad :(
ok, i'll unify